### PR TITLE
fix: add info and ticker view nft query

### PR DESCRIPTION
**Pull request by phucledien**
**What does this PR do?**

- [x] move the listing price beside the marketplace
- [x] format listing price
- [x] add project info
- [x] add ticker view for nft (e.g neko #3400)

**How to test**

- `$nft mgc 1617`

**Media (Loom or gif)**
CleanShot 2022-10-11 at 16 49 18@2x
CleanShot 2022-10-11 at 16 49 29@2x
CleanShot 2022-10-11 at 16 49 36@2x

### CODE REVIEW

File: src/commands/community/nft/query.ts [hunk 1]
```diff
@@ -79,6 +101,294 @@ const txHistoryEmojiMap: Record<string, string> = {
...
+        chain
+      ),
+    ],
+  }
+}
+
+async function renderNftTickerChart(
+  data: ResponseIndexerNFTCollectionTickersData
+) {
+  if (!data?.tickers?.prices || !data?.tickers.times) {
```
Comment (anhnh12) [inline src/commands/community/nft/query.ts:220]:
```suggestion
  if (!data?.tickers?.prices || !data?.tickers?.times) {
```

File: src/commands/community/nft/query.ts [hunk 2]
```diff
@@ -79,6 +101,294 @@ const txHistoryEmojiMap: Record<string, string> = {
...
+      messageOptions = await fetchAndComposeNFTDetail(
+        msg,
+        symbol,
+        collectionAddress,
+        tokenId,
+        chain
+      )
+      break
+  }
+  await i.editReply({ ...messageOptions }).catch(() => null)
```
Comment (anhnh12) [inline src/commands/community/nft/query.ts:187]: do we need to check null?

**Review by anhnh12 — CHANGES_REQUESTED** [review]

File: src/commands/community/nft/query.ts [hunk 3]
```diff
@@ -79,6 +101,294 @@ const txHistoryEmojiMap: Record<string, string> = {
...
+    })
+    .on("collect", async (i) => {
+      await switchView(i, originMsg)
+    })
+    .on("end", () => {
+      msg.edit({ components: [] }).catch(() => null)
+    })
+}
+
+async function switchView(i: ButtonInteraction, msg: Message) {
```
Comment (anhnh12) [inline src/commands/community/nft/query.ts:152]: add i.deferUpdate() since we edit original message

**Review by anhnh12 — CHANGES_REQUESTED** [review]

File: src/commands/community/nft/query.ts (hunk 2 above)
Comment (phucledien) [reply src/commands/community/nft/query.ts:187]: no, since there is a default case

File: src/commands/community/nft/query.ts (hunk 3 above)
Comment (phucledien) [reply src/commands/community/nft/query.ts:152]: no need since I already call defer in the filter function

**Review by tuanddd — APPROVED** [review]

**Review by hoangharry — APPROVED** [review]

**Review by trkhoi — APPROVED** [review]

**Pull request closed without merging**